src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1519,6 +1519,10 @@ function ($a) {
                             ->defaultNull()
                             ->info('Transport name to send failed messages to (after all retries have failed).')
                         ->end()
+                        ->scalarNode('delayed_transport')
```

Comment:
I find the naming confusing. To me, it sounds like my message could be delayed no matter the transport used ([like this](https://symfony.com/doc/current/messenger.html#envelopes-stamps)) while this option actually brings a different strategy of handling the message: using the `kernel.terminate` event.

Maybe `handle_after_request` would be more suitable?